File: kotlin/brute-force/main_7568.kt
fun main() {
    val count = readLine()!!.toInt()
    val list = mutableListOf<Pair<Int, Int>>()
    for (i in 1..count) {
        val (x, y) = readLine()!!.split(' ').map { it.toInt() }
        list.add(x to y)
    }
    list.forEach { origin ->
        print("${list.count{ cmp ->
            origin.first < cmp.first && origin.second < cmp.second
        } + 1} ")
    }
}

File: cpp/brute-force/main_7568.cpp
#include <iostream>
#include <vector>
using namespace std;

int main() {
	ios_base::sync_with_stdio(false);
	cin.tie(nullptr);

	int n;
	cin >> n;
	vector<pair<int, int>> people(n);
	for (int i = 0; i < n; ++i) {
		int x, y;
		cin >> x >> y;
		people[i] = make_pair(x, y);
	}

	for (int i = 0; i < n; ++i) {
		int count = 0;
		for (int j = 0; j < n; ++j) {
			if (people[i].first < people[j].first && people[i].second < people[j].second) {
				count++;
			}
		}
		cout << count + 1 << '\n';
	}
}
